README.md

```diff
@@ -3,3 +3,4 @@
 ## Content
 - [macOS](/mac)
 - [unix](/unix)
+- [android](/docker)
```

Comment:
I already created that `unix` subfolder for this. I would like to structure this repo based on a system platform. I.e. `/unix/docker/...` WDYT?

Reply:
So please move it to the unix subfolder if you agree :)